rpcs3/Emu/RSX/Common/texture_cache_helpers.h

```diff
@@ -553,41 +554,50 @@ namespace rsx
 					ensure(attr.height == 1);
 				}
 
-				if ((surface_is_rop_target && g_cfg.video.strict_rendering_mode) ||
-					attr.width < surface_width ||
-					attr.height < surface_height ||
-					force_convert)
+				bool requires_processing = (attr.width != surface_width || attr.height != surface_height || force_convert);
+				if (surface_is_rop_target && g_cfg.video.strict_rendering_mode && texptr->samples() == 1)
```

Comment:
this can be optimized

Reply:
Rewritten. This version has incorrect safe-mode handling.